layouts/fastly/helix.vcl

```diff
@@ -271,6 +273,21 @@ sub vcl_recv {
       set req.backend = F_GitHub;
       set req.url = "/" + var.owner + "/" + var.repo + "/" + var.ref + "/" + re.group.1;
     }
+  } elseif (!req.http.Fastly-FF && req.http.Fastly-SSL && req.url.path ~ "__HLX\/([^\/]+)\/([^\/]+)\/([^\/]+)\/([^\/]+)\/(.*)\/DIST__\/(.*)") {
+    # This is the GitHub/OpenWhisk bundler. In this branch of the if statment the __HLX…DIST__ indicates that
```

Comment:
where is `__HLX`  and `DIST__` set? where does it come from ?

Reply:
From here: https://github.com/adobe/project-helix/pull/266/files#diff-eaa78a59d4b66e15b887023e5b31b4a4R108

I'm using this pattern because it can be used anywhere in the URL, not just in the root path. For instance if you are defining strains based on the path, `/dist` (or `/static`) won't work. These URLs will most frequently be used for assets that are linked from an entry file, so as a developer you don't see them very often.

Reply:
iirgs...this is very ugly. another reason not to like this approach :-)

Reply:
I've added two items:

> - [ ] hide parts of the URL that can be retrieved from the strain (
https://github.com/adobe/helix-cli/pull/112#discussion_r212419875)
> - [ ] make `__HLX…DIST__` a random string (
https://github.com/adobe/helix-cli/pull/112#discussion_r212419875)

Reply:
> hide parts of the URL that can be retrieved from the strain
so, basically, just use `/dist` :)

Reply:
More like `$dist`, where `$dist` does not start with a slash so that it works everywhere.

Reply:
>  does not start with a slash so that it works everywhere.

why would that be required?

Reply:
If your strain is mapped to a path instead of a subdomain going back to `/dist` would reset the strain. By enabling any root path for the `dist` directory you have more flexibility.